### PR TITLE
WIP: Release notes

### PR DESCRIPTION
**Pull request by arianvp**
This is currently Markdown formatted; as I was copying things from the
wire-server release notes.

NOTE:  2.103.0 is missing from CI but it is needed

(fill in your PR description)

## Checklist:

Please tick the following before making your PR:

* [ ] I ran `make` on this branch to build the docs and there are **no WARNINGs**.
* [ ] After running `make`, I looked at the output by opening build/html/index.html in a browser and I'm satisfied with the rendering.

### CODE REVIEW

**Review by julialongtin — APPROVED** [review]

**Comment by arianvp** [comment]
@smatting do you want to have a look at the release notes; and change the instructions if neccesary? also shall I backlink to the wire-server CHANGELOG.md where possible?

File: src/release-notes.rst [hunk 1]
```diff
@@ -0,0 +1,377 @@
...
+---------------
+
+In case the ``elasticsearch-index-create`` job fails this document
+describes how to create a new index.
+
+The index that brig is using is defined at
+``brig.config.elasticsearch.index`` of the ``wire-server`` chart. We
+will refer to its current setting as ``<OLD_INDEX>``.
+
+1. Choose a new index name that is different from ``<OLD_INDEX>``. We
```
Comment (arianvp) [inline src/release-notes.rst:155]: @smatting just checking to be sure.  these changes _only_ need to be done if the helm chart failed to do with update for whatever reason?

Comment (smatting) [reply src/release-notes.rst:152]: yes, exactly

**Comment by smatting** [comment]
> @smatting do you want to have a look at the release notes; and change the instructions if neccesary?

I updated the release notes

> also shall I backlink to the wire-server CHANGELOG.md where possible?

Yes. I'll also update the changes to the CHANGELOG.md in wire-server

**Pull request merged**